django/contrib/admindocs/locale/en/LC_MESSAGES/django.po

```diff
@@ -4,356 +4,252 @@ msgid ""
 msgid "Back to Model documentation"
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/model_index.html:18
 msgid "Model documentation"
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/model_index.html:43
 msgid "Model groups"
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/template_detail.html:8
 msgid "Templates"
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/template_detail.html:13
 #, python-format
 msgid "Template: %(name)s"
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/template_detail.html:16
 #, python-format
 msgid "Template: <q>%(name)s</q>"
 msgstr ""
 
 #. Translators: Search is not a verb here, it qualifies path (a search path)
-#: contrib/admindocs/templates/admin_doc/template_detail.html:19
 #, python-format
 msgid "Search path for template <q>%(name)s</q>:"
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/template_detail.html:22
 msgid "(does not exist)"
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/template_detail.html:26
 msgid "Back to Documentation"
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/template_filter_index.html:12
 msgid "Template filters"
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/template_filter_index.html:16
 msgid "Template filter documentation"
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/template_filter_index.html:22
-#: contrib/admindocs/templates/admin_doc/template_filter_index.html:43
 msgid "Built-in filters"
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/template_filter_index.html:23
 #, python-format
 msgid ""
 "To use these filters, put <code>%(code)s</code> in your template before "
 "using the filter."
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/template_tag_index.html:12
 msgid "Template tags"
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/template_tag_index.html:16
 msgid "Template tag documentation"
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/template_tag_index.html:22
-#: contrib/admindocs/templates/admin_doc/template_tag_index.html:43
 msgid "Built-in tags"
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/template_tag_index.html:23
 #, python-format
 msgid ""
 "To use these tags, put <code>%(code)s</code> in your template before using "
 "the tag."
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/view_detail.html:12
 #, python-format
 msgid "View: %(name)s"
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/view_detail.html:23
 msgid "Context:"
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/view_detail.html:28
 msgid "Templates:"
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/view_detail.html:32
 msgid "Back to View documentation"
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/view_index.html:16
 msgid "View documentation"
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/view_index.html:22
 msgid "Jump to namespace"
 msgstr ""
 
-#: contrib/admindocs/templates/admin_doc/view_index.html:27
```

Comment:
Why are the comments all removed?

Reply:
It's the style of the .pos distributed with Django, almost all catalogs were generated with the "--no-location" option in makemesssages. This wasn't, and as its plural forms were updated and  the file was being updatetd, I removed them to make it like the others. Those are comments with the places where string has been extracted, it doesn't affect the translations, no need to distribute them.